setup.cfg

```diff
@@ -61,6 +61,7 @@ net =
   tqdm>=4.32.1
   zeep>=3.4.0
 timeseries =
+  cdflib>=0.3.20
```

Comment:
Need to check this is a suitable version to pin to, and add a corresponding entry to the oldesttestdeps CI.